tests/integration/distributed/helpers.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+from typing import Optional, Dict
+
+import json
+import requests
+
+
+def call_api(method: str, url: str, payload: Optional[Dict] = None, headers: Dict = {'Content-Type': 'application/json'}):
+    return getattr(requests, method)(
+        url, data=json.dumps(payload), headers=headers
+    ).json()
+
+
+def get_results(query: str, url: str = 'http://0.0.0.0:45678/api/search', method: str = 'post', top_k: int = 10):
+    return call_api(
+        method=method,
+        url=url,
+        payload={'top_k': top_k, 'data': [query]},
+    )
+
+
+def send_flow(flow_yaml: str, pod_dir: Optional[str] = None, url: str = 'http://localhost:8000/v1/flow/yaml'):
+    pymodules_files = []
+    uses_files = []
+
+    if pod_dir is not None:
+        uses_files = [
+            ('uses_files', open(file_path)) for file_path in Path(pod_dir).glob('*.yml')
```

Comment:
Can we open these files in `ExitStack`? Sample code [here](https://github.com/jina-ai/jina/blob/master/jina/peapods/runtimes/remote/jinad/api.py#L129)

Reply:
Sure I will try

Reply:
Done @deepankarm